microsoft/testsuites/vm_extensions/vmsnapshot_extension.py

```diff
@@ -41,7 +41,7 @@ class VmSnapsotLinuxBVTExtension(TestSuite):
         Validate response of the restore point for validity.
         Attempt it a few items to rule out cases when VM is under changes.
         """,
-        priority=1,
+        priority=6,
```

Comment:
why change this?

Reply:
I was testing locally and wanted my test to go first, we can ignore this change.